e2e/test/scenarios/visualizations-tabular/smartscalar-trend.cy.spec.js

```diff
@@ -158,32 +183,6 @@ describe("scenarios > visualizations > trend chart (SmartScalar)", () => {
     cy.findByTestId("scalar-container").findByText("Woah: 68’800.0000% ! cool");
   });
 
-  it("should have data settings disabled if only one option to choose from", () => {
```

Comment:
With "previous value" and "static number" I don't think we can end up with a single option anymore, can't we?